test/common/run-tests

```diff
@@ -96,6 +96,7 @@ def finish_test(opts, test, affected_tests):
     """
 
     affected = any([test.command[0].endswith(t) for t in affected_tests])
+    affected = True
```

Comment:
argh :/ and I was wondering how the hack I managed to break those tests so badly... All tests were retried. (that goes for local hacks and `git add -u`)